packages/tokens/src/color.yaml

```diff
@@ -192,19 +192,14 @@ color:
     feedback:
       primary:
         value: '{ color.gray.800.value }'
-        dark: '{ color.gray.800.value }'
       error:
-        value: '{ color.system.red.200.value }'
-        dark: '{ color.system.red.900.value }'
+        value: hsla(0, 79%, 57.1%, 0.15)
       info:
-        value: '{ color.blue.200.value }'
-        dark: '{ color.blue.900.value }'
+        value: hsla(231.5, 100%, 62.5%, 0.15)
       success:
-        value: '{ color.system.green.200.value }'
-        dark: '{ color.system.green.900.value }'
+        value: hsla(153.9, 100%, 42.7%, 0.15)
       warning:
-        value: '{ color.system.yellow.200.value }'
-        dark: '{ color.system.yellow.900.value }'
+        value: hsla(45.2, 83.7%, 59.2%, 0.15)
```

Comment:
These are temporary and should ideally consume the system 500 colors and add alpha to those. As a followup PR I'd like to explore using Style Dictionary's [transitive transforms](https://github.com/amzn/style-dictionary/blob/main/examples/advanced/transitive-transforms/README.md) with chromajs to improve the versatility of our token primitives